src/Symfony/Component/Mime/Part/TextPart.php

```diff
@@ -220,13 +220,13 @@ private function getEncoder(): ContentEncoderInterface
         return self::$encoders[$this->encoding];
     }
 
-    public static function addEncoder(string $name, ContentEncoderInterface $encoder): void
+    public static function addEncoder(ContentEncoderInterface $encoder): void
```

Comment:
But this way it is a BC break, so we need to keep the parameter and throw a proper deprecation

Reply:
The method has been introduced in 7.2, isn't it?
https://github.com/symfony/symfony/pull/54975

Reply:
this is our last chance to remove that parameter, before releasing it in a RC release.

Reply:
Fine in this case